src/test/java/org/broadinstitute/hellbender/engine/VariantWalkerBaseUnitTest.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.engine;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class VariantWalkerBaseUnitTest extends CommandLineProgramTest {
+    final String baseVariants = packageRootTestDir + "engine/feature_data_source_test.vcf";
+
+    private static class TestVariantWalker extends VariantWalker {
```

Comment:
This needs a `@CommandLineProgramProperties` annotation. Use `TestProgramGroup.class` for the `programGroup` attribute.

Reply:
ok